apps/kg/train.py

```diff
@@ -30,8 +30,10 @@ def __init__(self):
                           help='root path of all dataset')
         self.add_argument('--dataset', type=str, default='FB15k',
                           help='dataset name, under data_path')
-        self.add_argument('--format', type=str, default='1',
+        self.add_argument('--format', type=str, default='built_in',
```

Comment:
do we support the same format as eval.py?

Reply:
Same for eval.py